pom.xml

```diff
@@ -51,6 +51,11 @@
     </plugins>
   </build>
   <dependencies>
+    <dependency>
```

Comment:
Please also include new azure-core to make sure the maven build successfully after dependency upgrade. Thanks.

Reply:
I have moved the service bud first in the list. So that will bring the  azure-core . So we do not need to add it.